qserv-deploy.sh

```diff
@@ -100,6 +100,7 @@ DOT_KUBE_DIR="$QSERV_CFG_DIR/dot-kube"
 mkdir -p "$DOT_KUBE_DIR"
 MOUNTS="$MOUNTS -v $DOT_KUBE_DIR:$CONTAINER_HOME/.kube"
 
+MOUNTS="$MOUNTS -v $DIR/homefs/.bashrc:$CONTAINER_HOME/.bashrc"
```

Comment:
Where does `homefs` path come from?  Is this universal?

Reply:
The qserv-deploy container mount host directory homefs/ in its homedir. Comment added.